DXRModules/DeusEx/Classes/DXRReplaceActors.uc

```diff
@@ -46,7 +46,7 @@ function ReplaceActors()
         else if( #var(prefix)WHPiano(a) != None ) {
             ReplacePiano(#var(prefix)WHPiano(a));
         }
-        else if( #var(prefix)MissionEndgame(a) != None && !#defined(revision) && !#defined(hx) ) {
+        else if( #var(prefix)MissionEndgame(a) != None && !#defined(hx) ) {
```

Comment:
This needs to be Revision-aware and spawn a version subclassed from RevisionMissionEndgame instead if it's using Revision *and* Revision maps.  If it isn't using Revision Maps, it needs to use the one subclassed from Vanilla MissionEndgame.

Reply:
Or only do this replacement on Endgame4 (since Vanilla Endgame4 has no mission script, and the Revision one does, so you could choose to only ever replace with the Revision version